manifests/02-namespace.yaml

```diff
@@ -29,3 +31,4 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
     openshift.io/node-selector: ""
+    workload.openshift.io/allowed: "management"
```

Comment:
We're going to add a CI job to require all of our namespaces to have the annotation, and it would be easier to add the value here than track the special case in the CI test job definition.

Including the annotation won't trigger any extra behaviors if there are never any pods in the namespace.

Reply:
Thanks for the clearing things up!